_posts/0000-01-02-AzeemZ.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Hello,
```

Comment:
"Every fool can write code that machines can understand. Good programmers write code that humans can understand."